### DISCUSSION TITLE
Using arrays in Rowy

### DISCUSSION THREAD

**Discussion by harinij** [AntlerVC/firetable#731] [Support / Q&A] [answered]
To use arrays in Rowy, you can use the `json` field and under the column config, make sure the "default to array" option is on.

https://user-images.githubusercontent.com/307298/181876842-c029c4a1-7bfd-43d4-b556-822fdbab65c4.mov

**Comment by harinij** [comment]
This question was an information post and answered in the video above.

**Comment by davidoort** [comment]
Hi @harinij, I've tried to follow this structure for creating a list of document references, but when I do this in firebase, I can't see the data in rowy and in fact the app crashes when I click in the respective column. 
I'm happy to share a screen recording if you need more info, but maybe you're already aware of this and working on supporting **lists of doc references** which I personally use very frequently in Firebase

**Reply by ortsuev** [reply]
Hello @davidoort ! Have you solved this problem with the list of document links? I'm stuck at this point :(

**Comment by harinij** [comment, marked as answer]
HI, @davidoort @ortsuev we now have a new array field type that can have a list of doc references. Hopefully, this resolves your issue.

Documentation: https://docs.rowy.io/field-types/array
Video: https://www.youtube.com/watch?v=-_zQjsIwPx8
